client/reader/conversations/controller.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import route from 'lib/route';
+import { recordTrack } from 'reader/stats';
+import { renderWithReduxStore } from 'lib/react-helpers';
+import AsyncLoad from 'components/async-load';
+import { trackPageLoad } from 'reader/controller-helper';
+
+const ANALYTICS_PAGE_TITLE = 'Reader';
+
+export function conversations( context ) {
+	const basePath = route.sectionify( context.path );
+	const fullAnalyticsPageTitle = ANALYTICS_PAGE_TITLE + ' > Conversations';
```

Comment:
what the point of this convention.

can we just make a locally scoped `analyticsPageTitle = Reader > Conversations`?